packages/bsky/src/db/migrations/20230802T232331476Z-native-notifications.ts

```diff
@@ -0,0 +1,17 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('notification_push_token')
```

Comment:
we'll probably want to split endpoints & tokens into different tables at some point - I think I'm inclined to do so now to avoid a future migration 🤔

Reply:
hmmm actually, i don't think we even need endpoint yet if we're not doing 3rd party push notifs